Common/File/PathBrowser.cpp

```diff
@@ -210,7 +210,7 @@ std::string PathBrowser::GetFriendlyPath() const {
 bool PathBrowser::GetListing(std::vector<File::FileInfo> &fileInfo, const char *filter, bool *cancel) {
 	std::unique_lock<std::mutex> guard(pendingLock_);
 	while (!IsListingReady() && (!cancel || !*cancel)) {
-		// In case cancel changes, just sleep.
+		// In case cancel changes, just sleep. TODO: Replace with condition variable.
```

Comment:
Just remember to actually consider cancellation.  It might make sense to have a cancellable object that notifies multiple waiters (in this case the flag is used by multiple things.)

-[Unknown]

Reply:
Yeah, good point.